features/guidance/collapse.feature

```diff
@@ -995,13 +996,13 @@ Feature: Collapse
             | restriction | ph       | hi     | h        | no_right_turn |
 
         When I route I should get
-            | waypoints | route          | turns                       |
-            | a,e       | hohe,hohe      | depart,arrive               |
-            | a,s       | hohe,a100,a100 | depart,on ramp left,arrive  |
-            | a,t       | hohe,a100,a100 | depart,on ramp right,arrive |
-            | a,j       |                |                             |
-            | f,j       | hohe,hohe      | depart,arrive               |
-            | a,t       | hohe,a100,a100 | depart,on ramp right,arrive |
-            | f,e       |                |                             |
-            | q,j       | a100,hohe,hohe | depart,turn right,arrive    |
-            | q,e       | a100,a100,hohe | depart,continue left,arrive |
+            | waypoints | route                 | turns                       |
+            | a,e       | hohe,hohe             | depart,arrive               |
+            | a,s       | hohe,a100,a100        | depart,on ramp left,arrive  |
+            | a,t       | hohe,a100,a100        | depart,on ramp right,arrive |
+            | a,j       |                       |                             |
+            | f,j       | hohe,hohe             | depart,arrive               |
+            | a,t       | hohe,a100,a100        | depart,on ramp right,arrive |
+            | f,e       |                       |                             |
+            | q,j       | a100,hohe,hohe        | depart,turn right,arrive    |
+            | q,e       | a100,hohebruecke,hohe | depart,turn left,arrive     |
```

Comment:
With the feature in, can you add `locations` to the test and check the turn locations? For the respective change, locations should be `q,p,e`